### DISCUSSION TITLE
OGSR GUNSLINGER QUICK FEEDBACK

### DISCUSSION THREAD

**Discussion by ramseysparrow** [OGSR/OGSR-Engine#463] [Ideas]
First: OGRS + Gunslinger is the best Stalker experience I have ever tried. You guys, with all the work of others along the way, NAILED IT. It looks, walks and talks the way you _remember_ SoC to be, but with all the polish and focused refinement it always needed.

Everyone will have their own personal preferences and ideas, but there are 6 big points that stand out to me immediately, which I think are quite objectively based:

1. Crosshair. Please allow disabling the classic circle crosshair when weapons are holstered. I know OGSR goes for the classic stalker feel, but other than for nostalgia reasons, that big ugly circle is just jarring and unnecessary.

2. HUD. Same as above, I understand the SoC vibe, but the hud is GIANT and really distracting. Please consider offering a simple alternative option that minimises it to the modern standards (Stalker 2 is perfect example) and make the minimap optional or changed to a simple linear compass at the centre top.

3. Allow to initiate sprinting directly from crouched position. Currently it's blocked.

4. Allow switching off completely or controlling the level of zoom from weapon aiming.

5. Make the 3d map section buttons bigger or allow for toggling between the tabs with shortcuts (ie: 'Q' & 'E')

6. There are a lot of controls, a press/hold control binding system would help (ie: tap the key for 'x', hold the key for 'y')

7. Give option to make stalker identification work via binoculars (not enemy outlining and tagging, just a center identifier like the one currently tied to the crosshair)

8. Zombies. These guys eat a completely unreasonable amount of bullets right now, compared to everything else. Maybe normalise their bullet sponginess, or make a specific tactic against them more clear (ie: blast the limbs off or some such).

 Thank you for this mod, It bought the Stalker back into  Stalker for me, what a fun ride.

**Comment by ramseysparrow** [comment]
ps. forgot to mention one other thing: the high-pitched ringing effect from grenades is a straight route to permanent hearing damage for people playing on headphones (which is probably just about anyone playing stalker): it's way above the safe decibel levels compared to the global mixing level of the game. There's no benefit to keeping it anywhere as laud as it is at present - it does not increase immersion in any way, and can be safely reduced by half and then some.

**Comment by I-AmDead** [comment]
Interesting feedback and suggested elements, but it's unlikely that all this is being implemented, to switch tabs at least you need to make edits in the interface for this (unfortunately, there are no ideas for them yet)

I'll probably go through the points:

1. It's a great idea to remove the crosshair when there is nothing in your hands
2. As for where, there are ideas to introduce it into any gaming device, both hood and inventory
3. Definitely not, it will not be difficult to get out of the situation and start a sprint
4. May be
5. Great idea, but as I said earlier, you need to come up with an interesting interface
6. What is this for?
7. Well, that's interesting, too.
8. There is a tactic for killing zombies, you knock him to the ground by shooting him in the head and already on the ground you cut his head

**Reply by ramseysparrow** [reply]
Hey there. Are you one of the devs? Nice to hear from you, if so,  and congrats on a really great mod. 


1. Great. My point about binoculars identification (no 7) would be the simplest way to compensate for loosing the crosshair as means of identifying stalkers and it would make logical sense too (you're zooming-in on character details and faces to identify them). 

2. I honestly think hud is the most jarring thing about the mod right now, because it's so much in your face. Even if you're planning more advanced features for later, I'd definitely prioritise giving the hud a minimal facelift asap as an alternative version from vanilla.

3. Your mod, your call. For what it's worth, I feel if the player presses the sprint key the character would not think 'i need to stand straight first', he'd just spring up into a sprint, automatically cancelling the toggled crouching, maybe with a slightest delay. I think blocking it completely feels a bit artificial.

5. Not quite sure what you mean here.

6. Stalker has a lot of key binds and OGSR GU adds even more. I'm suggesting coding-in a feature where the game engine understands tapping and holding down of a key as different inputs. That way you could cut the necessary keybinds by half and let Players cluster things into logical groups, for example: 

   [tap]   [Y] = flashlight   /   [hold] [Y] = night vision
   [tap]   [R] = reload        /   [hold] [R] = change ammo
   [tap]   [G] = grenade    /   [hold] [G] = grenade launcher
   [tap]   [1]  = weapon 1  /   [hold] [1]  = use quick item 1
   [tap]   [U] = engine      /    [hold] [U] = add fuel


8. Haha... ok, I get it, I get it. Still, constantly switching between guns and knives for a swarm of zombies will probably get old pretty soon. In any case, it may need some thinking over as it does feel a bit weird right now. 


Thanks again for a great mod to you and the team, and look forward to updates and addons.
